src/app/components/Footer/index.tsx

```diff
@@ -53,23 +87,27 @@ export const Footer = memo(() => {
           i18nKey="footer.github"
           t={t}
           components={{ GithubLink: <Anchor href={githubLink} target="_blank" rel="noopener noreferrer" /> }}
-          defaults="ROSE Wallet is fully <GithubLink>open source</GithubLink> - Feedback and issues are appreciated!"
-        />
-      </Text>
-      <Text size={responsiveSize} textAlign="center" margin={{ bottom: responsiveSize }}>
-        <Trans
-          i18nKey="footer.terms"
-          t={t}
-          components={{
-            TermsLink: (
-              <Anchor href="https://wallet.oasis.io/t-c.pdf" target="_blank" rel="noopener noreferrer" />
-            ),
-          }}
-          defaults="<TermsLink>Terms and Conditions</TermsLink>"
+          defaults="ROSE Wallet is fully <GithubLink>open source</GithubLink> - and powered by
+          Oasis Scan API & Oasis gRPC"
```

Comment:
Oh and we prob want to keep dynamic "backend label" too. Like we plan to support Nexus in the future so we can switch between backend APIs when one is having issues.